charms/worker/k8s/charmcraft.yaml

```diff
@@ -161,6 +161,32 @@ config:
         CIDR to use for Kubernetes services. After deployment it is
         only possible to increase the size of the IP range. It is not possible to
         change or shrink the address range after deployment.
+    local_storage_enabled:
+      type: boolean
+      description: |
+        Enable local storage provisioning. This will create a storage class
+        named "local-storage" that uses the hostPath provisioner. This is
+        useful for development and testing purposes. It is not recommended for
+        production use.
+    local_storage_local_path:
+      type: string
+      default: ""
+      description: |
+        The path on the host where local storage will be provisioned. This
+        path must be writable by the kubelet. This is only used if
+        local-storage.enabled is set to true.
```

Comment:
```suggestion
        storage_local_enabled is set to true.
```

Reply:
As written in MM, I think `storage_local` is kinda weird. Is there a reason you want to  switch the order?

Reply:
as agreed on in the stand-up: `local-storage` it is